build/grunt.js

```diff
@@ -405,6 +410,14 @@ module.exports = function(grunt) {
         'watch:babel',
         'browserify:tests'
       ],
+      dev: [
+        'browserify:watch',
+        'browserify:tests',
+        'watch:novtt',
+        'watch:skin',
+        'watch:dist',
+        'shell:babel'
```

Comment:
we run `shell:babel` to use babel's built-in file watching. `grunt-babel` provides watching using grunt's `watch` module that we use above, however, it recompiles *all* the files instead of the files that were changed.